src/main/java/lib/choreolib/ChoreoTrajectory.java

```diff
@@ -70,4 +72,13 @@ public double getTotalTime() {
   public Pose2d[] getPoses() {
     return states.stream().map((state)->state.getPose()).toArray(Pose2d[]::new);
   }
+
+  public ChoreoTrajectory mirrorred() {

```

Comment:
Please fix spelling on `mirrored`. Also do we want to use `mirrored` in one place and `flipped` in another?